### PR TITLE
Feature/issue17 test server

### LINKED ISSUE #17: Server response should contain ordered list of all plants
## Description
The development server needs to be updated to match latest specifications:

- [x] Server poll response needs to include full list of plants to be displayed.
- [x] `has_error` attribute needs to be replaced with a status enumeration indicating the status of the data download process.

## Additional context
See comments for 2021/02/18 and 2021/02/20 in design doc.

### PR DESCRIPTION
**Pull request by power-guard**
## Description
Updates test server to latest response format. Includes ordered list of all power plants.
Includes additional use-case examples (parallel processing of data downloads, data download error scenario).

**Note:** Endpoint for the test server has been modified to better reflect the real usage. Requests will now need to be sent to `localhost:23500/api/dashboard/poll`

## Related Issue
Resolves #17

### CODE REVIEW

**Review by jcmtallon — APPROVED** [review]
LGTM!
With your permission, I am going to merge this branch in a few minutes so I can rebase it and work on top of it. Thanks!

File: srv/pg_monitor_demo_srv/dashboard/worker.py [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+}
+
+
+solar_edge_site2 = {
+    'energy': {
+        'timeUnit': 'QUARTER_OF_AN_HOUR',
+        'unit': 'Wh',
+        'measuredBy': 'INVERTER',
+        'values': [
+            {'date': '2021-02-20 00:00:00', 'value': None},
```
Comment (jcmtallon) [inline srv/pg_monitor_demo_srv/dashboard/worker.py:430]: [Question] Would it be correct that I display `none` values (received as `null` values in the API response) as `0.0` values? I haven't tried passing this data to the charts yet. Might be the case the charts parse these values correctly without me having to parse the data first. 

Should the front-end display some kind of error in this case, or receiving `null` values is expected behaviour? 

P.S.: Thinking more about it, I guess that is precisely one of the roles of the ERROR and WARNING states in the logs.

File: srv/README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 # Demo server
 
-## Getting started
+## Getting started (Docker Compose)
+
+1. From inside the `/srv` directory, run Docker Compose as follows:
+
+```shell
+docker-compose up
+```
```
Comment (jcmtallon) [inline srv/README.md:9]: Thanks for setting up `docker-compose` too. It is no doubt one step more convenient in this way.

File: srv/pg_monitor_demo_srv/dashboard/worker.py (hunk 1 above)
Comment (power-guard) [reply srv/pg_monitor_demo_srv/dashboard/worker.py:430]: Receiving `null` is expected behavior, as it represents that there is no data available for that time. Typically, this represents the solar power generation equipment being turned off.
For the purposes of the display, they can be safely shown as `0.0`.

**Pull request merged**